FIX-#3311: Fix part of warnings in docs build (#3325)
modin-project/modin: docs/flow/modin/backends/index.rst

''''''''''''''''''''''''''
This module houses submodules of all of the stable query compilers:

..

>     TODO: Insert link for <cuDF module> when it will be added (issue #3323)

Reviewer: ```suggestion
    TODO: Insert a link to <cuDF module> when it is added (issue #3323)
```

Author: applied

- :doc:`Base module <base/query_compiler>` contains an abstract query compiler class which defines common API.
- :doc:`Pandas module <pandas/index>` contains query compiler and text parsers for pandas backend.

> - cuDF module contains query compiler and text parsers for cuDF backend.

Reviewer: Let's add TODO not to forget to insert a link to cuDF module after we document it.

Author: TODO was added above
